src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -310,6 +325,18 @@ public void Stop()
         }
 
         #region private 
+        private void NotifyError(Exception exception)
+        {
+            // Throw and catch the exception.  This lets it be seen by the debugger
+            // ETW, and other monitoring tools.   However we immediately swallow the
+            // exception.   We may wish in the future to allow users to hook this 
+            // in other useful ways but for now we simply swallow the exceptions.  
+            try
+            {
```

Comment:
My only concern with that this is whether the extra dependency is bad (I debated having a Debugger.Break() to make it very obvious).      Ultimately I decided can add any of these things in the future, so it is not critical that it be put in now.    

Since exceptions do show up in the Output window (frankly more prominently than Debug.Write() statements), I think it is OK to be minimalist to start off.    I am happy to change this if we find that it would be helpful (and we don't mind the dependency).